yoti_python_sdk/dynamic_sharing_service/policy/wanted_attribute_builder.py

```diff
@@ -47,6 +47,8 @@ def build(self):
         """
         :return: The wanted attribute object
         """
+        assert self.__attribute["name"] is not None
+        assert self.__attribute["name"] != ""
```

Comment:
I don't think we should be using assert in non-test code.  Something like a `ValueError` would be better, as it's something that a user could recover from.  `assert` is generally used to describe something that should never happen and is non-recoverable.